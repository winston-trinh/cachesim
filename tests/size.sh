./csim -S 1 -K 1 -B 2 -p LRU -t traces/simple_size.trace
./csim -S 1 -K 1 -B 4 -p LRU -t traces/simple_size.trace
./csim -S 1 -K 1 -B 8 -p LRU -t traces/simple_size.trace
./csim -S 1 -K 2 -B 2 -p LRU -t traces/simple_size.trace
./csim -S 1 -K 2 -B 4 -p LRU -t traces/simple_size.trace
./csim -S 1 -K 2 -B 8 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 1 -B 2 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 1 -B 4 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 1 -B 8 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 2 -B 2 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 3 -B 2 -p LRU -t traces/simple_size.trace
./csim -S 2 -K 4 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 1 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 1 -B 4 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 1 -B 8 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 2 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 2 -B 4 -p FIFO -t traces/simple_size.trace
./csim -S 1 -K 2 -B 8 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 1 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 1 -B 4 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 1 -B 8 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 2 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 3 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 2 -K 4 -B 2 -p FIFO -t traces/simple_size.trace
./csim -S 32 -K 2 -B 4 -p LRU -t traces/simple_size2.trace
./csim -S 32 -K 2 -B 4 -p LRU -t traces/fifo_m1.trace
./csim -S 8 -K 4 -B 4 -p LRU -t traces/fifo_m1.trace
./csim -S 32 -K 2 -B 4 -p LRU -t traces/fifo_m2.trace
./csim -S 8 -K 4 -B 4 -p LRU -t traces/fifo_m2.trace
./csim -S 16 -K 2 -B 16 -p LRU -t traces/fifo_l.trace
./csim -S 32 -K 2 -B 4 -p FIFO -t traces/fifo_m1.trace
./csim -S 8 -K 4 -B 4 -p FIFO -t traces/fifo_m1.trace
./csim -S 32 -K 2 -B 4 -p FIFO -t traces/fifo_m2.trace
./csim -S 8 -K 4 -B 4 -p FIFO -t traces/fifo_m2.trace
./csim -S 16 -K 2 -B 16 -p FIFO -t traces/fifo_l.trace
