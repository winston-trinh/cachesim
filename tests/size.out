hits:2 misses:6 evictions:5
hits:2 misses:4 evictions:3
hits:3 misses:3 evictions:2
hits:2 misses:6 evictions:4
hits:2 misses:4 evictions:2
hits:4 misses:2 evictions:0
hits:2 misses:6 evictions:4
hits:2 misses:4 evictions:2
hits:4 misses:2 evictions:0
hits:2 misses:6 evictions:2
hits:3 misses:5 evictions:0
hits:3 misses:5 evictions:0
hits:2 misses:6 evictions:5
hits:2 misses:4 evictions:3
hits:3 misses:3 evictions:2
hits:2 misses:6 evictions:4
hits:2 misses:4 evictions:2
hits:4 misses:2 evictions:0
hits:2 misses:6 evictions:4
hits:2 misses:4 evictions:2
hits:4 misses:2 evictions:0
hits:2 misses:6 evictions:2
hits:3 misses:5 evictions:0
hits:3 misses:5 evictions:0
hits:1 misses:2 evictions:0
hits:93 misses:636 evictions:572
hits:74 misses:655 evictions:623
hits:315 misses:322 evictions:258
hits:262 misses:375 evictions:343
hits:265938 misses:26127 evictions:26095
hits:88 misses:641 evictions:577
hits:70 misses:659 evictions:627
hits:309 misses:328 evictions:264
hits:249 misses:388 evictions:356
hits:263701 misses:28364 evictions:28332
